apps/console/src/features/identity-providers/components/forms/helpers/form-fields-helper.tsx

```diff
@@ -475,8 +498,12 @@ export const getFieldType = (propertyMetadata: CommonPluggableComponentMetaPrope
     } else if (propertyMetadata?.key === CommonConstants.SCOPE_KEY) {
         return FieldType.TABLE;
     } else if (propertyMetadata?.key.toUpperCase().includes(CommonConstants.FIELD_COMPONENT_KEYWORD_URL)) {
-        // todo Need proper backend support to identity URL fields.
-        return FieldType.URL;
+        if (propertyMetadata?.key === AUTHORIZATION_REDIRECT_URL) {
+            return  FieldType.COPY_INPUT;
+        } else {
+            // todo Need proper backend support to identity URL fields.
```

Comment:
Lets create a product-is issue and link here.
```suggestion
            // TODO: Need proper backend support to identity URL fields.
```